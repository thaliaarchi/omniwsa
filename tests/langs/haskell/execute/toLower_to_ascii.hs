import Data.Char (toLower, chr)

main :: IO ()
main = do
    let chars = [chr c | c <- [0x80..0x10FFFF], toLower (chr c) <= '\x7f']
    putStrLn "Characters which Data.Char.toLower maps to ASCII:"
    mapM_ (\ch -> putStrLn (ch : " -> " ++ [toLower ch])) chars
