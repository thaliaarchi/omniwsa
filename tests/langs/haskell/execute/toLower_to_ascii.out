The Glorious Glasgow Haskell Compilation System, version 9.8.1
Characters which Data.Char.toLower maps to ASCII:
İ -> i
K -> k
